int16[] agent_ids
---
geometry_msgs/Point[] persons
